apps/admin-gui-e2e/src/integration/group-management-perun_admin.spec.js

```diff
@@ -0,0 +1,241 @@
+    }
+  });
+
+  beforeEach(() => {
+    cy.visit('home')
+      .get(`[data-cy=${dbGroupName}]`)
+      .click();
+  });
+
+  // // for higher priv user testing
+  // before(() => {
+  //   if (Cypress.env('BA_USERNAME')) {
+  //     sessionStorage.setItem('baPrincipal', '{"name": "perun"}');
+  //     sessionStorage.setItem('basicUsername', Cypress.env('BA_USERNAME'));
+  //     sessionStorage.setItem('basicPassword', Cypress.env('BA_PASSWORD'));
+  //     cy.visit('service-access');
+  //   }
+  // });
+  //
+  // beforeEach(() => {
+  //   cy.visit('organizations')
+  //     .get(`[data-cy=${dbVoName}]`)
+  //     .click()
+  //     .get('[data-cy=groups]')
+  //     .click()
+  //     .get(`[data-cy=${dbGroupName}]`)
+  //     .click();
+  // });
+
+  it('test create subgroup', () => {
+
+    cy.intercept('**/groupsManager/createGroup/**').as('createGroup')
+      .intercept('**/groupsManager/getGroups').as('getGroups')
+      .get(`[data-cy=subgroups]`)
+        .click()
+      .get('[data-cy=new-subgroup-button]')
+        .click()
+      .get('[data-cy=group-name]')
+        .type('test-group')
+      .get('[data-cy=group-description]')
+        .type('test-group-description')
+      .get('[data-cy=create-group-button-dialog]')
+        .click()
+      .wait('@createGroup')
+      // assert that group was created
+      .get('[data-cy=test-group-checkbox]')
+      .should('exist')
+  });
+
+  it('test remove subgroup', () => {
+
+    cy.intercept('**/groupsManager/deleteGroups').as('deleteGroups')
+      .get('[data-cy=subgroups]')
+      .click()
+      .get(`[data-cy=test-group-checkbox]`)
+      .click()
+      .get('[data-cy=delete-group-button]')
+      .click()
+      .get('[data-cy=delete-button-dialog]')
+      .click()
+      .wait('@deleteGroups')
+      // assert that group was deleted
+      .get(`[data-cy=test-group-checkbox]`)
+      .should('not.exist')
+  });
+
+  it('test add attribute', () => {
+
+    cy.intercept('**/attributesManager/setAttributes/**').as('setAttributes')
+      .get('[data-cy=attributes]')
+        .click()
+      .get('[data-cy=add-attributes]')
+        .click()
+      .get('[data-cy=block-manual-member-adding-value]')
+        .click()
+      .get('[data-cy=save-selected-attributes]')
+        .click()
+      .wait('@setAttributes')
+      // assert that attribute exists
+      .get('[data-cy=block-manual-member-adding-value]')
+      .should('exist')
+  });
+
+  it('test delete attribute', () => {
+
+    cy.intercept('**/attributesManager/removeAttributes/**').as('removeAttributes')
+      .get('[data-cy=attributes]')
+        .click()
+      .get('[data-cy=block-manual-member-adding-value]')
+        .click()
+      .get('[data-cy=remove-attributes]')
+        .click()
+      .get('[data-cy=delete-attributes]')
+        .click()
+      .wait('@removeAttributes')
+      // assert that attribute exists
+      .get('[data-cy=block-manual-member-adding-checkbox]')
+      .should('not.exist')
+  });
```

Comment:
The same. We want to have the tests independent. 
(If you keep the attribute "blockManualMemberAdding" on this subgroup, you will not be able to add members to this subgroup, but it couldn't be a problem in this case - you are adding members to the parent group in the next test.)